styles/06-components/brand-theme.css

```diff
@@ -0,0 +1,91 @@
+ body {
+  color: var(--color-text);
+  background-color: var(--color-background);
+  min-height: 100vh;
+}
+
+.heading ,
+.text {
+  line-height: 1.75;
+  text-align: center;
+}
+
+.nav-site .form-item {
+  width: fit-content;
+  align-self: center;
+}
+input[type="search"].form-input  {
+  width:clamp(30ch, 40rem, 60ch);
+}
+
+.nav-site .list-inline{
+  margin: 0;
+}
+
+.grid-fill {
+  grid-template-columns: repeat(auto-fill, minmax(35rem, 1fr));
+}
+
+.wishlist header {
+  text-align: center;
+}
+
+.quantity{
+  display: flex;
+  flex-flow: row;
```

Comment:
Suggestion:
You can declare these files in util and use it as class , as it can be used in multiple areas.